### DISCUSSION TITLE
SageMath

### DISCUSSION THREAD

**Discussion by anarorujov** [dat-linux/community#13] [App, Extra, Book suggestions]
Looking forward the [SageMath](https://www.sagemath.org) to be added to the distro!

**Comment by dat-linux** [comment]
Thanks for the suggestion!

**Comment by anarorujov** [comment]
Please consider to add GNU Octave also.

**Reply by dat-linux** [reply]
The only option to get the latest Octave on Linux/Ubuntu 22.04 is Flatpack via Flathub.

**Reply by dat-linux** [reply]
But a new "tab" for Flatpack apps (either only as flatpacks such as Octave, or as alternative install method to other DS apps) is in the works.
